extensions/resource-deployment/src/services/tools/dockerTool.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+import { ToolType, ITool, ToolInstallationStatus } from '../../interfaces';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+
+export class DockerTool implements ITool {
+	get name(): string {
+		return 'docker';
+	}
+
+	get description(): string {
+		return localize('resourceDeployment.DockerDescription', 'Manages the containers');
```

Comment:
I assume these descriptions are a WIP as well? Some could definitely use a readability/clarity pass

Reply:
basically all the strings need to be reviewed by Dinakar.